src/gui/appclient.cpp

```diff
@@ -443,6 +443,14 @@ void AppClient::onSignalReceived(int id, /*SignalNum*/ int num, const QByteArray
             emit showSynthesisDialog();
             break;
         }
+        case SIGNAL_NUM_UTILITY_LOG_UPLOAD_STATUS_UPDATED: {
+            char status;  // See types.h -> AppStateKey for the possible values of status
```

Comment:
Initialization

Reply:
Thanks, I will  fix this in the same time I will replace status by an enum.